Pipfile

```diff
@@ -61,6 +62,7 @@ pydenticon = "==0.3"
 pyotp = "==2.3.0"
 qrcode = "==6.1"
 requests = "==2.21.0"
+sendgrid-django = "==4.2.0"
```

Comment:
this still should not be in the Pipfile - the only place where it's needed is in `release.txt` along with our other deployment-only dependencies